src/UI/Component/Popover/Popover.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace ILIAS\UI\Component\Popover;
+
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Signal;
+use ILIAS\UI\Component\Triggerable;
+use ILIAS\UI\Implementation\Component\Popover\ReplaceContentSignal;
+
+/**
+ * Describes the Popover component
+ */
+interface Popover extends Component, Triggerable {
+
+	/**
+	 * Get the title of the popover.
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+	/**
+	 * Get the components representing the content of the popover.
+	 *
+	 * @return Component[]
+	 */
+	public function getContent();
+
+	/**
+	 * Get the position of the popover.
+	 *
+	 * @return string
```

Comment:
You would only need to have one of these, since it's either horizontal or vertical. You could also just add the constants. I can also currently see no harm in only implementing the getter on the concrete implementation of the component. The renderer acts on a concrete implementation, correct? Or does it need to consider every possible implementation? Not sure atm, mental load is too high in the JF.